spec/support/webmock.rb

```diff
@@ -1,3 +1,3 @@
 require 'webmock/rspec'
 
-WebMock.disable_net_connect!
+WebMock.disable_net_connect!(allow_localhost: true, allow: 'example.com')
```

Comment:
- preco potrebujeme chodit na localhost?
- preco potrebujeme chodit na example.com?

```suggestion
WebMock.disable_net_connect!(allow_localhost: true)

```

Reply:
- example.com nepotrebujeme, odstranil som
- localhost potrebuje capybara pri js: true testoch